### PR TITLE
Adds extradimensional oranges

### PR DESCRIPTION
**Pull request by Lett1**
## About The Pull Request

Adds a new mutation for oranges: 3D oranges!

orange rotate

https://streamable.com/893p7

## Why It's Good For The Game

orange rotate

## Changelog
:cl:
add: Adds a new mutation to the orange
/:cl:

### CODE REVIEW

**Comment by Lett1** [comment]
Now with 100% less merge conflicts!

**Comment by Fikou** [comment]
give it mindbreaker toxin

**Comment by tralezab** [comment]
distilling into triple sec is actually pretty funny

**Comment by duckay** [comment]
now add a wumpa fruit

**Comment by Tlaltecuhtli** [comment]
is this 3d asset subject to copyrights?

**Comment by mrhugo13** [comment]
I aM AbSoLuTeLy DiSgUsTeD By ThIs ThReE DiMeNsIoNaLe ObJeCt BeInG DaReD To Pr In My SpEsHuL 2D SpEsS mAn GaMe, It'S ExIsTaNcE PoLlUtInG ThIs WoRlD. My ImMeRsIoN HaS bEeN dEsTrOyEd aNd My DaY hAs bEeN RuInEd.

**Comment by Whoneedspacee** [comment]
is this enter the gungeon

**Comment by Farquaar** [comment]
Stats?

**Comment by necromanceranne** [comment]
How deep into the reference rabbit hole can this thing go?

**Comment by Barhandar** [comment]
Do they splatter into 3D gibs if thrown with liquid contents added?

**Comment by Lett1** [comment]
> is this 3d asset subject to copyrights?

I made it myself in blender, completely original.

> Do they splatter into 3D gibs if thrown with liquid contents added?

No

**Comment by nervere** [comment]
No player references.

**Comment by necromanceranne** [comment]
Don't worry, nervere, haven't you heard? Oranges doesn't even play.

**Comment by vuonojenmustaturska** [comment]
man has not fallen. he was never noble to begin with. if we had not committed great sins, god would not have sent punishment like this upon us. it is what we deserve. it is what we are owed. this disgusting extradimensional orange

**Comment by Partheo** [comment]
@nervere This is a reference to God Hand, not the player... It's just an orange.

**Comment by dr-bee** [comment]
I Love It!

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -157,3 +157,39 @@
...
+	potency = 20
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_grow = "lime-grow"
+	icon_dead = "lime-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	reagents_add = list("vitamin" = 0.04, "nutriment" = 0.05)
+
+/obj/item/reagent_containers/food/snacks/grown/citrus/orange_3d
+	seed = /obj/item/seeds/orange
+	name = "extradminesional orange"
```
Comment (Angustmeta) [inline code/modules/hydroponics/grown/citrus.dm]: 2 typos in same word.

Comment (Lett1) [reply code/modules/hydroponics/grown/citrus.dm]: Oh ffs, thanks for spotting that

**Pull request merged**